pygmt/datasets/load_remote_dataset.py

```diff
@@ -409,15 +428,20 @@ def _load_remote_dataset(
             f"'region' is required for {dataset.description} resolution '{resolution}'."
         )
 
-    # Currently, only grids are supported. Will support images in the future.
-    kwdict = {"T": "g", "R": region}  # region can be None
+    kwdict = {"R": region}  # region can be None
+    if name in {"earth_day"}:
+        kind = "image"
+        kwdict.update({"T": "i"})
+    else:
+        kind = "grid"
+        kwdict.update({"T": "g"})
```

Comment:
Maybe rewrite these lines to:
```suggestion
    kind = "image" if name in {"earth_day"} else "grid"
    kwdict["T"] = "i" if kind == "image" else "g"
```

Reply:
Done at 9a5e36bde20d4648196fc3bf5d0e4d37cbc3968b